zipline/gens/tradesimulation.py

```diff
@@ -356,9 +341,9 @@ def on_dt_changed(self, dt):
         if self.algo.datetime != dt:
             self.algo.on_dt_changed(dt)
 
-    def get_message(self, dt):
+    def generate_messages(self, dt):
         """
-        Get a perf message for the given datetime.
+        Generator that yields perf messages for the given datetime.
```

Comment:
Remind me again of the benefit of making this a generator? It's not immediately clear to me.